src/Fixer/PhpUnit/PhpUnitOrderedAnnotationsFixer.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\PhpUnit;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\DocBlock\DocBlock;
+use PhpCsFixer\Fixer\ConfigurationDefinitionFixerInterface;
+use PhpCsFixer\FixerConfiguration\AllowedValueSubset;
+use PhpCsFixer\FixerConfiguration\FixerConfigurationResolver;
+use PhpCsFixer\FixerConfiguration\FixerOptionBuilder;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Preg;
+use PhpCsFixer\Tokenizer\Token;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ * @author Andreas Möller <am@localheinz.com>
+ */
+final class PhpUnitOrderedAnnotationsFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getDefinition()
+    {
+        return new FixerDefinition(
+            'Order annotations of PHPUnit tests.',
+            [
+                new CodeSample(
+                    '<?php
+/**
+ * @uses Foo
+ * @uses Bar
+ */
+final class MyTest extends \PHPUnit_Framework_TestCase
+{}
+',
+                    [
+                        'annotations' => [
+                            'uses',
+                        ],
+                    ]
+                ),
+            ]
+        );
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
Do we really need these DocBlocks?

Reply:
We don't but I would leave it for consistency with the project. Maybe we'll revise this with #4500.